source/extensions/common/wasm/wasm.cc

```diff
@@ -1371,24 +1371,31 @@ void Context::setMetadataStruct(MetadataType type, absl::string_view name,
 }
 
 void Context::scriptLog(spdlog::level::level_enum level, absl::string_view message) {
+  std::string id;
+  if (!wasm()->id().empty()) {
+    id = id + " " + std::string(wasm()->id());
+  }
+  if (!root_id().empty()) {
+    id = id + " " + std::string(root_id());
+  }
```

Comment:
Could this be done inside constructor instead of each `log` call?

Reply:
Yes.  Done.